tools/depsgen/mkfile.go

```diff
@@ -55,15 +55,60 @@ _DEPS_GEN_SPACE_ :=
 _DEPS_GEN_F1_ :=
 _DEPS_GEN_F2_ :=
 `
+	kvStoredPrefix  = "_DEPS_GEN_STORED_"
```

Comment:
We should add a newline over this one. The backquote is hard to see and people can think it's part of the same variable